### PR TITLE
Allow delete command to unassign weddings

### PR DESCRIPTION
**Pull request by yikjunxian**
weddings can be unassigned from persons using:
- delete INDEX/NAME w/1 w/2

will add testcases asap

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -36,37 +43,53 @@ public class DeleteCommand extends Command {
...
+        if (personToDelete == null) {
+            return new CommandResult(String.format(MESSAGE_DUPLICATE_HANDLING));
+        }
+
+        if (isDeleteWedding) {
+            checkValidWeddingIndices(model);
+            // check if person was assigned to those weddings
+            checkPersonIsAssignedWeddings(personToDelete, model);
+            // delete those weddings
+            removeWeddingJobs(personToDelete, model);
```
Comment (Droas590) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:85]: nice abstraction

**Review by Droas590 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -26,8 +27,14 @@ public class DeleteCommand extends Command {
             + "Parameters: INDEX (must be a positive integer) or KEYWORD (the name of contact)\n"
             + "Example: " + COMMAND_WORD + " 1 or " + COMMAND_WORD + " alex";
```
Comment (SherniceSng) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:28]: Need to update the command format

**Review by SherniceSng — COMMENTED** [review]
LGTM. Just need to update the command format here, in UG and DG

**Pull request merged**